mobile/ios/Interface.m

```diff
@@ -0,0 +1,28 @@
+//
+//  Interface.m
+//  EntangledKit
+//
+//  Created by Rajiv Shah on 4/13/18.
+//  Copyright © 2018 IOTA Foundation. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "Interface.h"
+
+#include "common/helpers/sign.h"
+#include "common/helpers/pow.h"
+
+char* doPOW(const char* trytes, int mwm) {
```

Comment:
Why do you even need the interface? the helpers from `common/helpers/*` are already providing these methods.